web/recv.js

```diff
@@ -0,0 +1,401 @@
+var Sink = function () {
+
+  var _fountainBuff = undefined;
+  var _errBuff = undefined;
+
+  // public interface
+  return {
+    on_decode: function (buff) {
+      if (_fountainBuff === undefined) {
+        const size = Module._cimbard_get_bufsize(); // max length of buff. We could also resize as we go...
+        const dataPtr = Module._malloc(size);
+        _fountainBuff = new Uint8Array(Module.HEAPU8.buffer, dataPtr, size);
+      }
+      if (buff.length == 0) { // sanity check
+        return;
+      }
+      _fountainBuff.set(buff);
+
+      console.log('sink decode ' + _fountainBuff); //TODO: base64?
```

Comment:
Some of these log messages are pretty verbose. Will probably clean this up in a follow up PR